File: CS/NumPadForm.cs
using System.Linq;
using System.Windows.Forms;
using DevExpress.LookAndFeel;
using DevExpress.XtraEditors;

namespace KeysInteractionSample {
    public partial class NumPadForm : XtraForm {
        SvgImageItem keysGroup;
        public NumPadForm() {
            InitializeComponent();
            keysGroup = svgImageBox1.FindItemById("keys");
            foreach(var key in keysGroup.Items) {
                var keyText = key.FindDescendantById($"txt_{key.Id}");
                keyText.Text = key.Id.ToUpper();
                keyText.Appearance.Hovered.FillColor = DXSkinColors.IconColors.White;
                keyText.Appearance.Pressed.FillColor = DXSkinColors.IconColors.White;
                key.Appearance.Hovered.FillColor = DXSkinColors.IconColors.Blue;
                key.Appearance.Pressed.FillColor = DXSkinColors.IconColors.Black;
            }
        }
        void OnQueryUniqueItemId(object sender, QueryUniqueItemIdEventArgs e) {
            e.Id = $"{e.Id}_{e.UseElementId}";
        }
        void OnQueryHoveredItem(object sender, SvgImageQueryHoveredItemEventArgs e) {
            // Hover the entire key item
            e.HoveredItem = GetKeyItem(e.HitItems.FirstOrDefault());
        }
        void OnItemPress(object sender, SvgImageItemEventArgs e) {
            /* do something */
            SendKeys.Send(e.Item.Id);
        }
        SvgImageItem GetKeyItem(SvgImageItem item) {
            if(item == null)
                return null;
            if(item.IsGroup && item.Parent == keysGroup)
                return item;
            return item?.FindAncestor(x => x.IsGroup && x.Parent == keysGroup);
        }
    }
}

File: VB/NumPadForm.vb
Imports System.Linq
Imports System.Windows.Forms
Imports DevExpress.LookAndFeel
Imports DevExpress.XtraEditors

Public Class NumPadForm
    Private keysGroup As SvgImageItem
    Public Sub New()
        InitializeComponent()
        keysGroup = svgImageBox1.FindItemById("keys")
        For Each key In keysGroup.Items
            Dim keyText = key.FindDescendantById($"txt_{key.Id}")
            keyText.Text = key.Id.ToUpper()
            keyText.Appearance.Hovered.FillColor = DXSkinColors.IconColors.White
            keyText.Appearance.Pressed.FillColor = DXSkinColors.IconColors.White
            key.Appearance.Hovered.FillColor = DXSkinColors.IconColors.Blue
            key.Appearance.Pressed.FillColor = DXSkinColors.IconColors.Black
        Next
    End Sub
    Sub OnQueryUniqueItemId(ByVal sender As Object, ByVal e As QueryUniqueItemIdEventArgs) Handles svgImageBox1.QueryUniqueItemId
        e.Id = $"{e.Id}_{e.UseElementId}"
    End Sub
    Sub OnQueryHoveredItem(ByVal sender As Object, ByVal e As SvgImageQueryHoveredItemEventArgs) Handles svgImageBox1.QueryHoveredItem
        ' Hover the entire key item
        e.HoveredItem = GetKeyItem(e.HitItems.FirstOrDefault())
    End Sub
    Sub OnItemPress(ByVal sender As Object, ByVal e As SvgImageItemEventArgs) Handles svgImageBox1.ItemPress
        ' do something 
        SendKeys.Send(e.Item.Id)
    End Sub
    Function GetKeyItem(ByVal item As SvgImageItem) As SvgImageItem
        If item Is Nothing Then Return Nothing
        If item.IsGroup AndAlso item.Parent Is keysGroup Then Return item
        Return item?.FindAncestor(Function(x) x.IsGroup AndAlso x.Parent Is keysGroup)
    End Function
End Class